wasm/src/algorithms/poseidon/mod.rs

```diff
@@ -116,4 +119,82 @@ mod tests {
             assert_eq!(hash_8_many, native_hash_8_many);
         }
     }
+
+    #[wasm_bindgen_test]
+    fn test_wasm_literal_psd_hashes_equal_native_hashes() {
+        // Create all instances of exported hasher structs.
+        let poseidon2 = Poseidon2::new();
+        let poseidon4 = Poseidon4::new();
+        let poseidon8 = Poseidon8::new();
+
+        // Create native instances of hasher structs.
+        let native_poseidon2 = Poseidon2Native::setup("AleoPoseidon2").unwrap();
+        let native_poseidon4 = Poseidon4Native::setup("AleoPoseidon4").unwrap();
+        let native_poseidon8 = Poseidon8Native::setup("AleoPoseidon8").unwrap();
+
+        let address = Address::zero();
+
+        let literals = vec![
+            Literal::Address(address),
+            Literal::Boolean(Boolean::new(false)),
```

Comment:
```suggestion
```

We don't export Boolean, so no need to test it. However, we do export Signature, so we should test that.